Loudspeaker_Explorer.md

```diff
@@ -2146,8 +2146,8 @@ The following [box plot](https://en.wikipedia.org/wiki/Box_plot) should be read
 
 A mathematically equivalent, and perhaps more useful, way of looking at this plot is the following:
 
-- If **boxes barely overlap** (predicted score differs by 1.0), there is a **83%** chance that the average listener will prefer the higher-rated speaker.
-- If **lines (whiskers) barely overlap** (predicted score differs by 1.8), the probability is **95%**.
+- If **boxes barely overlap** (predicted score differs by 1.08), there is a **83%** chance that the average listener will prefer the higher-rated speaker.
+- If **lines (whiskers) barely overlap** (predicted score differs by 1.84), the probability is **95%**.
```

Comment:
Can you rebase-squash (fixup) this commit into the "Fixed preference probability calculation and google colab check" commit?

Reply:
Done, let me know if there is anything else that needs to be done